ocw_data_parser/ocw_data_parser_test.py

```diff
@@ -444,6 +444,33 @@ def test_course_pages(ocw_parser):
     )
 
 
+def test_instructor_insights_divided_sections(ocw_parser_course_2):
```

Comment:
Can we assert the text somewhere? If we can't do the whole thing in a clean way, we should at least check that some of the content of two separate sections is included in the text